basic-javascript/project-2.spec.js

```diff
@@ -1,3 +1,270 @@
 const funcs = require('./project-2');
 
-// whoops.. there is no test suite for this file. You'll simply just have to create one :/
+it('two numbers input, returns largest number', () => {
+    const expected = 10;
+                                                
+    const actual = funcs.getBiggest(3, 10);
+                                                
+    expect(actual).toEqual(expected);
+});
+
+describe('greeting', () => {
+
+it('if greeting is string "German"', () => {
```

Comment:
Indent every `it` that belongs to the `describe` to make the code more readable.